setup.py

```diff
@@ -26,7 +26,7 @@
 
 def _parse_changelog():
     """Get version from debian changelog and write it to gbp/version.py"""
-    with open("debian/changelog") as f:
+    with open("debian/changelog", encoding="utf-8") as f:
```

Comment:
makes sense but could you spit that into a separate commit since it's not directly related to packaging.

Reply:
Done